src/vm/gcenv.ee.cpp

```diff
@@ -386,7 +386,7 @@ bool GCToEEInterface::EnablePreemptiveGC()
 
     if (pThread)
     {
-        bToggleGC = !!pThread->PreemptiveGCDisabled();
+        bToggleGC = pThread->PreemptiveGCDisabled();
```

Comment:
Why not fix this the same way as in the sample?

Reply:
Done